arcade/examples/sprite_depth_cosine.py

```diff
@@ -23,21 +23,22 @@
 import arcade
 
 # All constants are in pixels
-WIDTH, HEIGHT = 1280, 720
+WINDOW_WIDTH, WINDOW_HEIGHT = 1280, 720
 
+WINDOW_TITLE = "Sprite Depth Testing Example w/ a Cosine Wave"
 NUM_SPRITES = 10
 
 SPRITE_X_START = 150
 SPRITE_X_STEP = 50
-SPRITE_Y = HEIGHT // 2
+SPRITE_Y = WINDOW_HEIGHT // 2
 
 DOT_SIZE = 10
 
 
-class MyGame(arcade.Window):
+class DepthExample(arcade.View):
```

Comment:
Same naming note as above. post 3.0, we could discuss a different convention, but for now...